.gitignore

```diff
@@ -44,6 +44,9 @@ typings/
 # Optional npm cache directory
 .npm
 
+# knexfile
+knexfile.js
```

Comment:
@nguyenaiden mentioned that it's to avoid changing the file because the only thing that might change in that file is setting debug from false to true as you're testing